goja_test.go

```diff
@@ -0,0 +1,1680 @@
+package main
+
```

Comment:
all the test cases is in this file.
If we want a quick go through. `assert.Panics` can be a trick keyword. // for quick search